packages/components/organisms/f-cookie-banner/test/specs/component/shared/f-cookieBanner-legacy.component.spec.js

```diff
@@ -0,0 +1,15 @@
+const CookieBanner = require('../../../../test-utils/component-objects/f-cookieBanner-legacy.component');
+const cookieBanner = new CookieBanner();
+import forEach from 'mocha-each';
+
+describe('Legacy - Multi-tenant - f-cookieBanner component tests', () => {
+    forEach([['gb'], ['au'], ['nz']])
```

Comment:
when it's only one item, I don't think we need to add sub-arrays - it should work with just `['gb', 'au', 'nz'] ` 😄

Reply:
We tried this and I don't _think_ it works, mocha-each will think it's a single test with three params!